script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -637,7 +637,7 @@ My reason for not going out into the city is more practical. Lately, there've be
 [sha:b9980c8fe4e133461b213a53dc5162032ccf721a]{
 -- Page 115, Offset 1460.
 -- 北口駅前……オフィス街とは反対側の繁華街で、夜は交通量も人通りも多い、不夜城のような地域だ。
-In front of the north gate... that's downtown opposite the office district. At night it's bustling with lots of people and traffic.
+In front of the north gate... that's downtown opposite the business district. At night it's bustling with lots of people and traffic.
```

Comment:
alternative suggestion: "office park"

Reply:
i have no preference between any so i'll leave it up to you

Reply:
@Volvo-hackangel